src/lib/pipeline/adt/chunk/index.js

```diff
@@ -105,6 +108,19 @@ class Chunk extends THREE.Mesh {
     return bit & this.holes;
   }
 
+  dispose() {
+    this.geometry.dispose();
+    this.material.dispose();
+
+    this.material.textures.forEach((texture) => {
+      TextureLoader.unload(texture.sourceFile);
+    });
+
+    this.material.alphaMaps.forEach((alphaMap) => {
+      alphaMap.dispose();
+    });
+  }
```

Comment:
Overriding `Material#dispose` could be an option here. We'd keep the material construction and clean-up in one place, we'd have to remember to call `super` though (https://github.com/mrdoob/three.js/blob/master/src/materials/Material.js#L284-L288)